com.cubrid.cubridmanager.core/src/com/cubrid/cubridmanager/core/common/ServerManager.java

```diff
@@ -28,14 +28,13 @@
 
 package com.cubrid.cubridmanager.core.common;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashMap;
 import java.util.Iterator;
 import java.util.List;
-import java.util.Map;
 
+import com.cubrid.common.ui.spi.model.CubridServer;
```

Comment:
Package `com.cubrid.common.ui` dependent the package `com.cubrid.cubridmanager.ui`.
So, as you added package `com.cubrid.common.ui` in here cause cyclic dependency.

Would you please fix this problem?